<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriageOptionsDlg</class>
 <widget class="QDialog" name="TriageOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="dialogButtonBox_ApplyCancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Blur</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxBlurThreshhold">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>17.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlurThreshhold_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShiftSize_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Shift Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxShiftSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>53</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>130</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Brightness</string>
   </property>
   <widget class="QLabel" name="labelBrightnessMin_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>26</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBrightnessMin_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>MAX</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxBrightnessMin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>23</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="value">
     <double>65.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxBrightnessMax">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>53</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="value">
     <double>180.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Triage Options</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>35</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxTriageImagesReport">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Generate Report</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriageImagesMove">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Check this box to move bad images into the subfolder &quot;MovedImages&quot; which is automatically created beneath the image folder.</string>
       </property>
       <property name="text">
        <string>Move Bad Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_TriageImages_SavePolylines">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Check this box to move bad images into the subfolder &quot;MovedImages&quot; which is automatically created beneath the image folder.</string>
       </property>
       <property name="text">
        <string>Save Polylines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_TriageImages_CorrectAlignment">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Check this box to move bad images into the subfolder &quot;MovedImages&quot; which is automatically created beneath the image folder.</string>
       </property>
       <property name="text">
        <string>Correct Alignment</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Image Transformation</string>
   </property>
   <widget class="QPushButton" name="pushButton_SelectReferenceImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Reference Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBrightnessMin_5">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>72</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation Threshold</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_TriageOptions_RotationThreshold">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>71</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.250000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox_ApplyCancel</sender>
   <signal>accepted()</signal>
   <receiver>TriageOptionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox_ApplyCancel</sender>
   <signal>rejected()</signal>
   <receiver>TriageOptionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
